tmt/steps/provision/testcloud.py

```diff
@@ -575,9 +587,9 @@ def start(self):
             raise ProvisionError(
                 f'Failed to boot testcloud instance ({error}).')
         self.guest = self._instance.get_ip()
-        self.port = self._instance.get_instance_port()
+        self.port = int(self._instance.get_instance_port())
         self.verbose('ip', self.guest, 'green')
-        self.verbose('port', self.port, 'green')
+        self.verbose('port', str(self.port), 'green')
```

Comment:
This `int()` and `str()` isn't pretty, but the [super class](https://github.com/teemtee/tmt/blob/main/tmt/steps/provision/__init__.py#L686) defines `self.port` as `int`, so what can we do here? :)

Reply:
Nothing, keep it like this. `self.port`  is indeed an integer, and `verbose()` needs the value to be `Optional[str]`, so you're done, I believe.

Reply:
Why not f"{self.port}" ? I find it nicer.

Reply:
I for one find that much uglier, sort of "misusing" string formatting.

Reply:
+1, I am also for `str` conversion